pkg/network/tracer/tracer.go

```diff
@@ -467,6 +467,7 @@ func (t *Tracer) getConnTelemetry(mapSize int) map[network.ConnTelemetryType]int
 
 	stats, err := t.getStats(stateStats)
 	if err != nil {
+		log.Debugf("error getting state stats: %s", err)
```

Comment:
note - having logs in hot-code-paths can leads to a very high cpu usage

Reply:
removed, thanks!

Reply:
unresolved just as an FYI - 

there are 4 different options you can take:
1. Keep the log line (if we don't see any cpu regression in the load test profiles) - as the code is not an hot-path or the error is very rare
2. use log.ShouldLog (like [here](https://github.com/DataDog/datadog-agent/blob/ea98ae0218b75a6d3ea65a94991b608d44dfb7c5/pkg/collector/python/memory.go#L81-L83))
3. Use [LogLimit](https://github.com/DataDog/datadog-agent/blob/ea98ae0218b75a6d3ea65a94991b608d44dfb7c5/pkg/network/protocols/http/statkeeper.go#L144-L146) and limit the number of logs we can write in a given period in time 
4. Remove the log

feel free to resolve again